tests/testDbLocal.py

```diff
@@ -49,6 +49,7 @@
 import tempfile
 import time
 import unittest
+import os.path
```

Comment:
This is not needed, `help(os.path)` explicitely says not to import `os.path`, importing `os` is all that is needed